packages/ui-components/package.json

```diff
@@ -55,6 +55,7 @@
 	},
 	"dependencies": {
 		"@floating-ui/dom": "^0.4.0",
+		"clsx": "^1.2.1",
```

Comment:
Do we really need this? Stencil has its own similar way of contacting class names.
See my suggestion below.

Reply:
I didn't know about that. Thank you :pray: I will remove it :muscle: